File: src/main/java/net/kenro/ji/jin/purescript/psi/impl/PSGuardImpl.java
package net.kenro.ji.jin.purescript.psi.impl;

import com.intellij.lang.ASTNode;
import com.intellij.psi.PsiElementVisitor;
import net.kenro.ji.jin.purescript.psi.PSGuard;
import net.kenro.ji.jin.purescript.psi.PSVisitor;
import org.jetbrains.annotations.NotNull;

public class PSGuardImpl extends PSPsiElement implements PSGuard {

    public PSGuardImpl(final ASTNode node) {
        super(node);
    }

}

File: src/main/java/net/kenro/ji/jin/purescript/psi/impl/PSGuardImpl.kt
package net.kenro.ji.jin.purescript.psi.impl;

import com.intellij.lang.ASTNode;
import net.kenro.ji.jin.purescript.psi.PSGuard;

public class PSGuardImpl extends PSPsiElement implements PSGuard {

    public PSGuardImpl(final ASTNode node) {
        super(node);
    }

}
